### DISCUSSION TITLE
File Not Found

### DISCUSSION THREAD

**Discussion by sifuhall** [ImageMagick/ImageMagick#6028] [Help]
### ImageMagick version

7.1.0-60 Q16-HDRI x64

### Operating system, version and so on

Windows 11

### Description

I am attempting to use magick on the command line to convert a pdf file but I get the error file not found.

Here is the command and output.

C:\Users\sifuh\Downloads\temp>magick 1.pdf 1.png
magick: FailedToExecuteCommand `"gswin64c.exe" -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 "-sDEVICE=pngalpha" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 "-r72x72" -dPrinted=false  "-sOutputFile=C:/Users/sifuh/AppData/Local/Temp/magick-GnpWRYvv3Wl5c1HOsVUca3Zd8W07UGuz%d" "-fC:/Users/sifuh/AppData/Local/Temp/magick-jlLfVnaI39LKvje_VqJjYATIA_qViowk" "-fC:/Users/sifuh/AppData/Local/Temp/magick-BapkN5Aca6oqqfGM2jyd-Nu9m694yhAD"' (The system cannot find the file specified.
) @ error/delegate.c/ExternalDelegateCommand/517.
magick: PDFDelegateFailed `The system cannot find the file specified.
' @ error/pdf.c/ReadPDFImage/716.

**Comment by dlemstra** [comment]
You will need to install Ghostscript to read `pdf` files.
